HeterogeneousCore/CUDACore/README.md

```diff
@@ -83,7 +83,8 @@ This page documents the CUDA integration within CMSSW
        stream must synchronize with the work queued on other CUDA
        streams (with CUDA events and `cudaStreamWaitEvent()`)
 4. Outside of `acquire()`/`produce()`, CUDA API functions may be
-   called only if `CUDAService::enabled()` returns `true`.
+   called only if `CUDAService` is available and `CUDAService::enabled()`
+   returns `true`.
```

Comment:
Do you mean something like
```suggestion
   returns `true`:
     ```c++
     edm::Service<CUDAInterface> cuda;
     if (cuda and cuda->enabled()) {
       // CUDA calls ca be made here
     }
     ```
```
?

Reply:
A code example is even better than what I had in mind.